unit-tests/live/frames/test-fps.py

```diff
@@ -66,14 +63,17 @@ def frame_cb(frame):
 
 delta_Hz = 1
 tested_fps = [6, 15, 30, 60, 90]
+time_to_test_fps = [20, 13, 10, 5, 4]
```

Comment:
I wonder how you chose those numbers :)
Maybe you can add a comment.

I tried following some logic that you ask for 300 frames but it's not persistent.

Reply:
No explicit rule. On one hand I wanted enough frames, around 300 looked like a good number. On the other hand I wanted to reduce the test time so on low FPS I settled for less frames.